LvivCompany.Bookstore/LvivCompany.Bookstore.DataAccess/Context/BookStoreContextFactory.cs

```diff
@@ -1,15 +1,15 @@
-﻿using Microsoft.EntityFrameworkCore;
```

Comment:
The code in this file is commented out. If it's not required, why not delete the file?